libs/libarchfpga/src/expr_eval.cpp

```diff
@@ -2,13 +2,17 @@
 #include "arch_error.h"
 #include "vtr_util.h"
 #include "vtr_math.h"
+
 #include <string>
 #include <sstream>
 
 using std::stack;
 using std::string;
 using std::stringstream;
 using std::vector;
+current_information current_info_e;
+bool is_breakpoint = false;
```

Comment:
The expression evaluator is general purpose code, so it likely shouldn't have any context specific behavior (such as whether it is evaluating a breakpoint or not).